faststream_gen/_components/embeddings.py

```diff
@@ -119,6 +124,82 @@ def _delete_directory(d: str) -> None:
             print(f"Error deleting directory: {e}")
 
 # %% ../../nbs/Embeddings_CLI.ipynb 11
+def _read_lines_from_file(file_path: Path, lines_spec: str) -> str:
```

Comment:
This was previously done in the faststream repo?

Reply:
Yes, we have to move these helper functions to this repo. So we can get delete the faststream_gen related files in Faststream repo